nano/core_test/fakes/work_peer.hpp

```diff
@@ -176,9 +185,7 @@ class work_peer_connection : public std::enable_shared_from_this<work_peer_conne
 		}
 		else if (action_text == "work_cancel")
 		{
-			error (empty_response);
-			on_cancel ();
-			write_response ();
+			handle_cancel ();
```

Comment:
This isn't being tested here which is fine, but to keep the current behavior, `write_response ()` should not be removed.

Reply:
It's all been delegated to the handle_cancel method instead, write_response is still there.

Reply:
Oops, seems you're right. I forgot to append it to the end.